packages/puppeteer-core/src/api/Browser.ts

```diff
@@ -286,44 +277,43 @@ export class Browser extends EventEmitter<BrowserEvents> {
   }
 
   /**
-   * Creates a new incognito browser context. This won't share cookies/cache with other
-   * browser contexts.
+   * Creates a new incognito {@link BrowserContext | browser context}.
+   *
+   * This won't share cookies/cache with other {@link BrowserContext | browser contexts}.
    *
    * @example
    *
    * ```ts
-   * (async () => {
-   *   const browser = await puppeteer.launch();
-   *   // Create a new incognito browser context.
-   *   const context = await browser.createIncognitoBrowserContext();
-   *   // Create a new page in a pristine context.
-   *   const page = await context.newPage();
-   *   // Do stuff
-   *   await page.goto('https://example.com');
-   * })();
+   * import puppeteer from 'puppeteer';
```

Comment:
Sadly you can't use top-level await in non-modules. 
Revert this as it if you set-up a Typescript from scratch you will get a lot of errors.